src/libs/recommendation/useHomeRecommendedIdsMutation.ts

```diff
@@ -19,14 +19,21 @@ export const useHomeRecommendedIdsMutation = () => {
           body: JSON.stringify(requestBodyParams),
         })
         if (!response.ok) {
-          throw new Error('Failed to fetch recommendation')
+          throw new Error('Failed to fetch recommendation (response not ok)')
         }
         const responseBody: RecommendedIdsResponse = await response.json()
+        if (!responseBody.playlist_recommended_offers.length) {
+          eventMonitoring.captureMessage('Recommended offers playlist is empty', {
+            level: 'info',
+            extra: { url: endpointUrl },
```

Comment:
question(non-blocking): est-ce que l'url nous suffira pour comprendre ce qui n'a pas été ?

Reply:
Dans ce cas je pensais que le message `Recommended offers playlist is empty` était assez claire, t'aurais ajouté quelque chose?

Reply:
je ne sais pas, c'est une question très ouverte

est-ce qu'on a des données dynamiques qu'on pourrait mettre dans `extra` pour nous aider ?